modules/core/src/main/java/org/apache/ignite/internal/visor/verify/CacheKind.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.visor.verify;
+
+import org.jetbrains.annotations.Nullable;
+
```

Comment:
Please check code style in this class.

Reply:
`CacheKind` isn't clear class name. Let's move enum inside `VisorIdleVerifyDumpTaskArg` or `VisorIdleVerifyDumpTask` and rename to `VisorVerifyDumpPartitionMode` or `VisorVerifyDumpPartitionOption`

Reply:
I have renamed the enum, that the name more clear, but does not moved to internal class, because I   does not see the need.